microsoft/testsuites/vm_extensions/azsecpack.py

```diff
@@ -123,7 +123,7 @@ def verify_azsecpack(self, node: Node, log: Logger, result: TestResult) -> None:
         # Install agent extension
         try:
             self._install_monitor_agent_extension(node)
-            self._install_security_agent_extension(node, log)
+            # self._install_security_agent_extension(node, log)
```

Comment:
1. Please explain the reason to comment out it in the PR.
2. Remove the line directly, instead of commenting out.

Reply:
This is to test a specific custom image. In order to run a pipeline I had to raise this draft PR and then use its GitHub reference in the pipeline. This is the reason I marked the PR as draft.